spec/facade/menus_facade_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe 'MenuFacade' do
+  subject(:menu_facade) { Menu::MenuFacade.new(create(:menu, :with_meals)) }
+
+  it 'still has a menu object' do
+    expect(menu_facade.menu).to be_a Menu
+  end
+
+  it 'can access methods from decorators' do
+    expect(menu_facade.menu.day_stamp).to be_a String
+  end
+
+  context 'when menu has meals' do
+    let(:meal) { menu_facade.meals.first }
```

Comment:
Tests have to be atomic meaning they shouldn't rely on the data from previous suits & specs. Use `DatabaseCleaner` configured with `:each` and `:suite` and refactor tests.